set.mm

```diff
@@ -647910,26 +647917,26 @@ elements and the subgroup containing only the identity ( ~ simpgnsgbid ).
     ablsimpgfindlem1.6 $e |- ( ph -> G e. SimpGrp ) $.
     $( Lemma for ~ ablsimpgfind .  An element of an abelian finite simple group
        which doesn't square to the identity has finite order.  (Contributed by
-       Rohan Ridenour, 3-Aug-2023.) $)
-    ablsimpgfindlem1 $p |- ( ( ( ph /\ x e. B ) /\ ( 2 .x. x ) =/= .0. ) ->
-      ( O ` x ) =/= 0 ) $=
-      ( wcel c2 co cc0 wceq cz c1 vy wne cfv w3a cabl 3ad2ant1 csimpg simpggrpd
-      vz cv cgrp 2z simp2 mulgcld simp3 neneqd ablsimpg1gend wa cmul cmin cdvds
-      a1i wbr simprr simpl2 mulg1 syl adantr mulgassr syl13anc 3eqtr4rd zmulcld
-      wn simprl 1zzd odcong syl112anc mpbird 0zd caddc 2t0e0 oveq1i 0p1e1 eqtri
-      wb zneo neeqtrd oveq1 syl6req adantl cc 2cnd zcn mulcld 1cnd npcan syl2an
-      eqtr2d ex necon3ad syl5 anabsi5 syl2anc zsubcld 0dvds mtbird rexlimddvcbv
-      nbrne2 eqeq2d 3expa ) ABUJZCNZOXKDPZGUBZXKFUCZQUBZAXLXNUDZXPXKUAUJZXMDPZR
-      ZXKUIUJZXMDPZRUIUASXQXMCXKDUIEGHIJAXLEUENXNLUFAXLEUGNXNMUFXQCDEOXKHJAXLEU
-      KNZXNAEMUHUFZOSNZXQULVBAXLXNUMZUNXQXMGAXLXNUOUPYFUQXQXRSNZXTURZURZXOOXRUS
-      PZTUTPZVAVCZQYKVAVCZVMXPYIYLYJXKDPZTXKDPZRZYIXKXSYOYNXQYGXTVDYIXLYOXKRAXL
-      XNYHVEZCDEXKHJVFVGYIYCYGYEXLYNXSRXQYCYHYDVHZXQYGXTVNZYEYIULVBZYQCDEXROXKH
-      JVIVJVKYIYCXLYJSNTSNYLYPWEYRYQYIOXRYTYSVLZYIVOZXKDEYJTFCGHKJIVPVQVRYIYMYK
-      QRZYIYGQSNZUUCVMZYSYIVSYGUUDUUEYGUUDURZYJTUBYGUUEUUFYJOQUSPZTVTPZTXRQWFUU
-      HTRUUFUUHQTVTPZTUUGQTVTWAWBWCWDVBWGYGUUCYJTYGUUCYJTRYGUUCURTYKTVTPZYJUUCT
-      UUJRYGUUCUUJUUITYKQTVTWHWCWIWJYGYJWKNTWKNUUJYJRUUCYGOXRYGWLXRWMWNUUCWOYJT
-      WPWQWRWSWTXAXBXCYIYKSNYMUUCWEYIYJTUUAUUBXDYKXEVGXFXOQYKVAXHXCYAXRRYBXSXKY
-      AXRXMDWHXIXGXJ $.
+       Rohan Ridenour, 3-Aug-2023.)  (Proof shortened by Rohan Ridenour,
+       31-Oct-2023.) $)
+    ablsimpgfindlem1 $p |- ( ( ( ph /\ x e. B ) /\ ( 2 .x. x ) =/= .0. ) -> ( O ` x ) =/= 0 ) $=
+      ( vy wcel co wne wceq 3ad2ant1 wa cv c2 cfv cc0 w3a cabl csimpg simpggrpd
```

Comment:
```suggestion
    ablsimpgfindlem1 $p |-
          ( ( ( ph /\ x e. B ) /\ ( 2 .x. x ) =/= .0. ) -> ( O ` x ) =/= 0 ) $=
      ( vy wcel co wne wceq 3ad2ant1 wa cv c2 cfv cc0 w3a cabl csimpg simpggrpd
```

Reply:
If someone can apply the modif and merge...

Reply:
For some reason this kind of github online changes do not work, but I could pull the PR and do the changes on my machine.